tests/Struct/FindologicProductTest.php

```diff
@@ -1542,19 +1556,77 @@ public function thumbnailProvider(): array
                         'highDpi' => false,
                         'url' => 'https://via.placeholder.com/400'
                     ]
+                ],
+                'expectedImages' => [
+                    'findologic.png'
+                ]
+            ],
+            '0 thumbnails, the automatically generated thumbnail is taken' => [
+                'thumbnails' => [],
+                'expectedImages' => [
+                    '600x600',
+                    '600x600'
                 ]
             ],
-            '0 thumbnails, the image directly assigned to the product is taken' => ['thumbnails' => []]
+            'Same thumbnail exists in various sizes will only export one size' => [
+                'thumbnails' => [
+                    [
+                        'width' => 800,
+                        'height' => 800,
+                        'highDpi' => false,
+                        'url' => 'https://via.placeholder.com/800'
+                    ],
+                    [
+                        'width' => 1000,
+                        'height' => 1000,
+                        'highDpi' => false,
+                        'url' => 'https://via.placeholder.com/1000'
+                    ],
+                    [
+                        'width' => 1200,
+                        'height' => 1200,
+                        'highDpi' => false,
+                        'url' => 'https://via.placeholder.com/1200'
+                    ],
+                    [
+                        'width' => 1400,
+                        'height' => 1400,
+                        'highDpi' => false,
+                        'url' => 'https://via.placeholder.com/1400'
+                    ],
+                    [
+                        'width' => 1600,
+                        'height' => 1600,
+                        'highDpi' => false,
+                        'url' => 'https://via.placeholder.com/1600'
+                    ],
+                    [
+                        'width' => 1800,
+                        'height' => 1800,
+                        'highDpi' => false,
+                        'url' => 'https://via.placeholder.com/1800'
+                    ],
+                ],
+                'expectedImages' => [
+                    '800x800',
+                    '800x800',
```

Comment:
So there is one actual image and one thumbnail, right? Can you make this more clear as they both have the same sizes.